### PR TITLE
chore: remove unnecessary suffix from @vaadin/flow-frontend imports

### PR DESCRIPTION
**Pull request by haijian-vaadin**
The flow-frontend has an [index.ts](https://github.com/vaadin/flow/blob/master/flow-client/src/main/resources/META-INF/resources/frontend/index.ts) file which exports the content from `Flow`, `Connect`, `Authentication` and `ConnectionIndicator`. So these suffixes are not necessary.

Related to https://github.com/vaadin/flow/issues/7419

### CODE REVIEW

File: articles/fusion/advanced/client-middleware.asciidoc [hunk 1]
```diff
@@ -96,16 +96,17 @@ import {MyLogMiddleware} from './my-log-middleware';
 client.middlewares = [MyLogMiddleware];
 ----
 
-NOTE: If you modify the middlewares array, only calls initiated after the modification will use the new middlewares array. To avoid issues related with that, it is better to avoid the modification of middlewares, or only modify the middlewares before the first call.
+NOTE: If you modify the middlewares array, only calls initiated after the modification use the new middlewares array.
```
Comment (jouni) [inline articles/fusion/advanced/client-middleware.asciidoc:99]:
```suggestion
.Modifying middleware at runtime
[CAUTION]
If you modify the middlewares array, only calls initiated after the modification use the new middlewares array. 
```

Always try to provide a summarizing title for admonitions (I’m not sure my suggestion is correct). Also, here I think "caution" could be used to give it more emphasis.

File: articles/fusion/application/basics.asciidoc [hunk 2]
```diff
@@ -206,14 +209,19 @@ export const router = new Router(document.querySelector('#outlet'));
 router.setRoutes(routes);
 ----
 
-Now `my-view` is accessible via the root path, i.e. `http://localhost:8080/`. All the other routes are handled by the server-side router. See the <<creating-routes#, Creating routes in TypeScript>> page for more information.
+Now `my-view` is accessible via the root path, that is `http://localhost:8080/`. All the other routes are handled by the server-side router.
```
Comment (jouni) [inline articles/fusion/application/basics.asciidoc:212]:
```suggestion
Now `my-view` is accessible via the root path, that is `http://localhost:8080/`.
All the other routes are handled by the server-side router. 
```

File: articles/fusion/application/basics.asciidoc [hunk 3]
```diff
@@ -206,14 +209,19 @@ export const router = new Router(document.querySelector('#outlet'));
...
 
-== Hot-reload in Development Mode
+== Hot Reload in Development Mode
 
-When running the application in development mode, all modifications in `frontend` folder are compiled automatically. Refreshing browser is enough to get the updates.
+When running the application in development mode, all modifications in `frontend` folder are compiled automatically.
+Refreshing browser is enough to get the updates.
 
-NOTE: There is an exceptional case when adding `index.ts` or `index.html` for the first time. The application server needs to be restarted to update the entry point and the bootstrap template.
+NOTE: When adding `index.ts` or `index.html` for the first time, the application server needs to be restarted to update the entry point and the bootstrap template.
```
Comment (jouni) [inline articles/fusion/application/basics.asciidoc:220]:
```suggestion
.Server restart is required
[NOTE]
When adding `index.ts` or `index.html`, the application server needs to be restarted to update the entry point and the bootstrap template.
```

I’m not sure the use of admonition/note here is needed. If you feel it is, please add a summarizing title.

I don’t “for the first time” is needed, as you can’t add those files more than once, can you? Or is there more nuance in this, like that if you first add `index.ts` and then restart the server, and if you then add `index.html` server restart is not needed?

File: articles/fusion/routing/nested-views.asciidoc [hunk 4]
```diff
@@ -90,7 +92,8 @@ import '@vaadin/vaadin-lumo-styles/all-imports';
 
 == Establish an Application Layout
 
-The most prominent feature of the main layout is to define the layout for the application. This could be done using the `<vaadin-app-layout>` component:
+The most prominent feature of the main layout is to define the layout for the application. 
+You can use the `<vaadin-app-layout>` component:
```
Comment (jouni) [inline articles/fusion/routing/nested-views.asciidoc:96]:
```suggestion
You can use the <<{articles}/ds/components/app-layout#,App Layout>> component:
```

File: articles/fusion/application/basics.asciidoc (hunk 3 above)
Comment (haijian-vaadin) [reply articles/fusion/application/basics.asciidoc:220]: very good observation! Thanks for the suggestion, I think it's better.

**Review by platosha — APPROVED** [review]

File: articles/fusion/advanced/typescript-client.asciidoc [hunk 5]
```diff
@@ -8,29 +8,30 @@ layout: page
...
+== Default TypeScript Client
 
 The default TypeScript client is the module that gets generated by the Vaadin maven plugin for a particular project.
 
 The module contains the `ConnectClient` implementation that is set up to communicate with the backend side of the project.
 
 == Generated Client Modules
 
-Along with the default TypeScript client, the Vaadin maven plugin is capable of generating specific modules for each Vaadin endpoint defined in the backend part. Those modules use the default TypeScript client described above to communicate with the backend also.
+Along with the default TypeScript client, the Vaadin maven plugin is capable of generating specific modules for each Vaadin endpoint defined in the backend part.
```
Comment (jouni) [inline articles/fusion/advanced/typescript-client.asciidoc:23]:
```suggestion
Along with the default TypeScript client, the Vaadin Maven plugin is capable of generating specific modules for each Vaadin endpoint defined in the backend part.
```

File: articles/fusion/application/basics.asciidoc [hunk 6]
```diff
@@ -206,14 +209,22 @@ export const router = new Router(document.querySelector('#outlet'));
...
+Now `my-view` is accessible via the root path, that is `http://localhost:8080/`.
+All the other routes are handled by the server-side router. 
+See the <<creating-routes#, Creating routes in TypeScript>> page for more information.
 
-== Hot-reload in Development Mode
+== Hot Reload in Development Mode
 
-When running the application in development mode, all modifications in `frontend` folder are compiled automatically. Refreshing browser is enough to get the updates.
+When running the application in development mode, all modifications in `frontend` folder are compiled automatically.
+Refreshing browser is enough to get the updates.
```
Comment (jouni) [inline articles/fusion/application/basics.asciidoc:219]:
```suggestion
Refreshing the browser is enough to see the updates in the application.
```

File: articles/fusion/application/basics.asciidoc [hunk 7]
```diff
@@ -206,14 +209,22 @@ export const router = new Router(document.querySelector('#outlet'));
...
+.Server restart is required
+[NOTE]
+When adding `index.ts` or `index.html`, the application server needs to be restarted to update the entry point and the bootstrap template.
 
 == Accessing Backend Data in TypeScript Views
 
-Vaadin 15 provides a type-safe and secured way to access data from backend in frontend views using the generated TypeScript code. During the development time, Vaadin scans the backend code and generates TypeScript code which can call corresponding Java methods. The generated code are processed through the same build chain as other TypeScript views. So that only necessary code are bundled for your application in production mode. See <<accessing-backend#, Accessing back-end from TypeScript>> for more information.
+Fusion provides a type-safe and secured way to access data from backend in frontend views using the generated TypeScript code. 
+During the development time, Vaadin scans the backend code and generates TypeScript code which can call corresponding Java methods.
```
Comment (jouni) [inline articles/fusion/application/basics.asciidoc:228]:
```suggestion
Vaadin scans the backend code during development and generates TypeScript code which can call the corresponding Java methods.
```

File: articles/fusion/application/basics.asciidoc [hunk 8]
```diff
@@ -206,14 +209,22 @@ export const router = new Router(document.querySelector('#outlet'));
...
+.Server restart is required
+[NOTE]
+When adding `index.ts` or `index.html`, the application server needs to be restarted to update the entry point and the bootstrap template.
 
 == Accessing Backend Data in TypeScript Views
 
-Vaadin 15 provides a type-safe and secured way to access data from backend in frontend views using the generated TypeScript code. During the development time, Vaadin scans the backend code and generates TypeScript code which can call corresponding Java methods. The generated code are processed through the same build chain as other TypeScript views. So that only necessary code are bundled for your application in production mode. See <<accessing-backend#, Accessing back-end from TypeScript>> for more information.
+Fusion provides a type-safe and secured way to access data from backend in frontend views using the generated TypeScript code. 
+During the development time, Vaadin scans the backend code and generates TypeScript code which can call corresponding Java methods. 
+The generated code are processed through the same build chain as other TypeScript views.
```
Comment (jouni) [inline articles/fusion/application/basics.asciidoc:229]:
```suggestion
The generated code is processed through the same way as other TypeScript views.
```

File: articles/fusion/application/basics.asciidoc [hunk 9]
```diff
@@ -206,14 +209,22 @@ export const router = new Router(document.querySelector('#outlet'));
...
 
 == Accessing Backend Data in TypeScript Views
 
-Vaadin 15 provides a type-safe and secured way to access data from backend in frontend views using the generated TypeScript code. During the development time, Vaadin scans the backend code and generates TypeScript code which can call corresponding Java methods. The generated code are processed through the same build chain as other TypeScript views. So that only necessary code are bundled for your application in production mode. See <<accessing-backend#, Accessing back-end from TypeScript>> for more information.
+Fusion provides a type-safe and secured way to access data from backend in frontend views using the generated TypeScript code. 
+During the development time, Vaadin scans the backend code and generates TypeScript code which can call corresponding Java methods. 
+The generated code are processed through the same build chain as other TypeScript views. 
+Only necessary code are bundled for your application in production mode. See <<accessing-backend#, Accessing backend from TypeScript>> for more information.
```
Comment (jouni) [inline articles/fusion/application/basics.asciidoc:230]:
```suggestion
Only the necessary code is included in the production application bundle. See <<accessing-backend#, Accessing backend from TypeScript>> for more information.
```

File: articles/fusion/application/starting.asciidoc [hunk 10]
```diff
@@ -31,15 +31,15 @@ router.setRoutes(serverSideRoutes);
 ----
 
 For applications where the UI is built in Java and does not include any client-side views the client-side bootstrapping process is transparent.
-A default `index.html` / `index.ts` pair is generated during the build, and immediately loads the server-side `UI` when the app starts.
-If there is a need to add a client-side view to the application later, it can be done incrementally, with minimal modifications.
+A default `index.html` / `index.ts` pair is generated during the build, and loads the server-side `UI` when the app starts.
```
Comment (jouni) [inline articles/fusion/application/starting.asciidoc:34]:
```suggestion
A default `index.html` / `index.ts` pair is generated during the build, and loads the server-side `UI` when the application starts.
```

File: articles/fusion/application/starting.asciidoc [hunk 11]
```diff
@@ -31,15 +31,15 @@ router.setRoutes(serverSideRoutes);
...
+If there is a need to add a client-side view to the application later, you can do it incrementally, with minimal modifications.
```
Comment (jouni) [inline articles/fusion/application/starting.asciidoc:35]:
```suggestion
Client-side views can be added incrementally to the application later, with minimal modifications.
```

File: articles/fusion/application/starting.asciidoc [hunk 12]
```diff
@@ -52,22 +52,31 @@ router.setRoutes([
 ----
 
 
-With client-side routing you can optimize your application to load the minimal content to be displayed to users at the first load. This might reduce the first-loading time compared to the Vaadin 10-14 bootstrapping method which loads a full Vaadin application on the first request. As a result, the first request loads promptly and that improves the first interaction experience.
+With client-side routing you can optimize your application to load the minimal content to be displayed to users at the first load. 
+This might reduce the first-loading time compared to the Vaadin 10–14 bootstrapping method which loads a full Vaadin application on the first request.
```
Comment (jouni) [inline articles/fusion/application/starting.asciidoc:56]:
```suggestion
Client-side routing can improve the first load/startup of the application, by minimizing the content delivered to the browser on initial load.
```
I don’t think we need to talk negatively about Vaadin 10–14 bootstrapping. Let’s just point out the benefit of using client-side routes.

File: articles/fusion/application/starting.asciidoc [hunk 13]
```diff
@@ -52,22 +52,31 @@ router.setRoutes([
 ----
 
 
-With client-side routing you can optimize your application to load the minimal content to be displayed to users at the first load. This might reduce the first-loading time compared to the Vaadin 10-14 bootstrapping method which loads a full Vaadin application on the first request. As a result, the first request loads promptly and that improves the first interaction experience.
+With client-side routing you can optimize your application to load the minimal content to be displayed to users at the first load. 
+This might reduce the first-loading time compared to the Vaadin 10–14 bootstrapping method which loads a full Vaadin application on the first request. 
+As a result, the first request loads promptly and that improves the first interaction experience.
```
Comment (jouni) [inline articles/fusion/application/starting.asciidoc:57]:
```suggestion
```
Redundant

File: articles/fusion/application/starting.asciidoc [hunk 14]
```diff
@@ -52,22 +52,31 @@ router.setRoutes([
...
 
 
 == Bootstrap Page Template [[bootstrap-page-template]]
 
-When using client-side bootstrapping, Vaadin servlet uses the `frontend/index.html` file as a template to generate the bootstrap page response. It processes the template and injects the following additional information:
+When using client-side bootstrapping, Vaadin servlet uses the `frontend/index.html` file as a template to generate the bootstrap page response.
```
Comment (jouni) [inline articles/fusion/application/starting.asciidoc:62]:
```suggestion
When using client-side bootstrapping, the Vaadin servlet uses the `frontend/index.html` file as a template to generate the bootstrap page response. 
```

File: articles/fusion/application/starting.asciidoc [hunk 15]
```diff
@@ -52,22 +52,31 @@ router.setRoutes([
...
+With client-side routing you can optimize your application to load the minimal content to be displayed to users at the first load. 
+This might reduce the first-loading time compared to the Vaadin 10–14 bootstrapping method which loads a full Vaadin application on the first request. 
+As a result, the first request loads promptly and that improves the first interaction experience.
 
 
 == Bootstrap Page Template [[bootstrap-page-template]]
 
-When using client-side bootstrapping, Vaadin servlet uses the `frontend/index.html` file as a template to generate the bootstrap page response. It processes the template and injects the following additional information:
+When using client-side bootstrapping, Vaadin servlet uses the `frontend/index.html` file as a template to generate the bootstrap page response. 
+It processes the template and injects the following information:
```
Comment (jouni) [inline articles/fusion/application/starting.asciidoc:63]:
```suggestion
The servlet processes the template and injects the following information:
```

File: articles/fusion/application/starting.asciidoc [hunk 16]
```diff
@@ -52,22 +52,31 @@ router.setRoutes([
...
 
-  - `<base href='./relative/to/root'>`: Vaadin calculates the relative path from the current request path to the root path of the application. That is required for relative links in the app view templates to work correctly.
+  - `<base href='./relative/to/root'>`: Vaadin calculates the relative path from the current request path to the root path of the application. 
+  That is required for relative links in the app view templates to work correctly.
```
Comment (jouni) [inline articles/fusion/application/starting.asciidoc:66]:
```suggestion
  That is required for relative links in view templates to work correctly.
```

File: articles/fusion/application/starting.asciidoc [hunk 17]
```diff
@@ -52,22 +52,31 @@ router.setRoutes([
...
 
-  - Bundled script: Vaadin automatically adds the bundled and optimized script generated from the `frontend/index.ts` file. It uses a pre-configured link:https://webpack.js.org/[Webpack^] instance that's included together with `vaadin-maven-plugin` as a module bundler. Therefore, in the `frontend/index.html` template there is no need to include the `index.ts` (or `index.js`) script manually.
+  - Bundled script: Vaadin automatically adds the bundled and optimized script generated from the `frontend/index.ts` file. 
+  It uses a pre-configured link:https://webpack.js.org/[Webpack^] instance that's included together with `vaadin-maven-plugin` as a module bundler.
```
Comment (jouni) [inline articles/fusion/application/starting.asciidoc:69]:
```suggestion
  It uses a pre-configured link:https://webpack.js.org/[webpack^] instance that's included together with `vaadin-maven-plugin` as a module bundler. 
```

File: articles/fusion/application/starting.asciidoc [hunk 18]
```diff
@@ -52,22 +52,31 @@ router.setRoutes([
...
+  - `<base href='./relative/to/root'>`: Vaadin calculates the relative path from the current request path to the root path of the application. 
+  That is required for relative links in the app view templates to work correctly.
 
-  - Bundled script: Vaadin automatically adds the bundled and optimized script generated from the `frontend/index.ts` file. It uses a pre-configured link:https://webpack.js.org/[Webpack^] instance that's included together with `vaadin-maven-plugin` as a module bundler. Therefore, in the `frontend/index.html` template there is no need to include the `index.ts` (or `index.js`) script manually.
+  - Bundled script: Vaadin automatically adds the bundled and optimized script generated from the `frontend/index.ts` file. 
+  It uses a pre-configured link:https://webpack.js.org/[Webpack^] instance that's included together with `vaadin-maven-plugin` as a module bundler. 
+  Therefore, in the `frontend/index.html` template there is no need to include the `index.ts` (or `index.js`) script manually.
```
Comment (jouni) [inline articles/fusion/application/starting.asciidoc:70]:
```suggestion
  Therefore, the `frontend/index.html` template does not need to include the `index.ts` (or `index.js`) script explicitly.
```

File: articles/fusion/application/starting.asciidoc [hunk 19]
```diff
@@ -52,22 +52,31 @@ router.setRoutes([
...
 
 NOTE: The frontend directory can be customized by providing the property `vaadin.frontend.frontend.folder` when running the Maven goals `prepare-frontend`  or `build-frontend` from `vaadin-maven-plugin`.
 
 === Default Bootstrap Template and Entry Point [[default-bootstrap-template-and-entry-point]]
 
-If the `index.html` or `index.ts` files in the frontend folder are missing, `vaadin-maven-plugin` generates a default corresponding file in the `target` folder. The generated files starts the application so that it uses only server-side routing. You can take control of these files by moving them into the `frontend` folder. By default these files look similar to the following:
+If the `index.html` or `index.ts` files in the frontend folder are missing, `vaadin-maven-plugin` generates a default corresponding file in the `target` folder. 
+The generated files starts the application so that it uses only server-side routing.
```
Comment (jouni) [inline articles/fusion/application/starting.asciidoc:77]:
```suggestion
In that case the application uses server-side routing only.
```

File: articles/fusion/application/starting.asciidoc [hunk 20]
```diff
@@ -118,4 +127,5 @@ export const router = new Router(document.querySelector('#outlet'));
 router.setRoutes(routes);
 ----
 
-NOTE: The auto-generated bootstrap template works best for server-side applicaitons. In order add client-side views into the app, please follow the  <<quick-start-guide#step-3, Quick Start Guide>>.
+NOTE: The auto-generated bootstrap template works best for server-side applicaitons.
```
Comment (jouni) [inline articles/fusion/application/starting.asciidoc:130]:
```suggestion
.The defaults are best for Flow applications
[NOTE]
The autogenerated bootstrap template works best for server-side applications. 
```

File: articles/fusion/application/starting.asciidoc [hunk 21]
```diff
@@ -118,4 +127,5 @@ export const router = new Router(document.querySelector('#outlet'));
 router.setRoutes(routes);
 ----
 
-NOTE: The auto-generated bootstrap template works best for server-side applicaitons. In order add client-side views into the app, please follow the  <<quick-start-guide#step-3, Quick Start Guide>>.
+NOTE: The auto-generated bootstrap template works best for server-side applicaitons. 
+To add client-side views into the app, please follow the  <<quick-start-guide#step-3, Quick Start Guide>>.
```
Comment (jouni) [inline articles/fusion/application/starting.asciidoc:131]:
```suggestion
To add client-side views into the application, follow the <<quick-start-guide#step-3, Quick Start Guide>>.
```

File: articles/fusion/routing/nested-views.asciidoc [hunk 22]
```diff
@@ -75,7 +75,9 @@ export const router = new Router(outlet);
 
 == Import Theme Global Styles
 
-In Vaadin, themes have some global styles that need to be applied to the page. With server-side main layout in Java, these are automatically applied for convenience. With TypeScript main layout, though, they need to be imported manually to make sure consistent styling between server-side and client-side views.
+In Vaadin, themes have some global styles that need to be applied to the page. 
+With server-side main layout in Java, these are automatically applied for convenience.
```
Comment (jouni) [inline articles/fusion/routing/nested-views.asciidoc:79]:
```suggestion
With a server-side main layout in Java, these are automatically applied for convenience. 
```

File: articles/fusion/routing/nested-views.asciidoc [hunk 23]
```diff
@@ -75,7 +75,9 @@ export const router = new Router(outlet);
 
 == Import Theme Global Styles
 
-In Vaadin, themes have some global styles that need to be applied to the page. With server-side main layout in Java, these are automatically applied for convenience. With TypeScript main layout, though, they need to be imported manually to make sure consistent styling between server-side and client-side views.
+In Vaadin, themes have some global styles that need to be applied to the page. 
+With server-side main layout in Java, these are automatically applied for convenience. 
+With TypeScript main layout, though, they need to be imported manually to make sure consistent styling between server-side and client-side views.
```
Comment (jouni) [inline articles/fusion/routing/nested-views.asciidoc:80]:
```suggestion
With a client-side main layout in TypeScript, they need to be imported manually to get consistent styling between server-side and client-side views.
```

**Review by jouni — CHANGES_REQUESTED** [review]

File: articles/fusion/application/starting.asciidoc (hunk 12 above)
Comment (haijian-vaadin) [reply articles/fusion/application/starting.asciidoc:56]: good point

**Review by platosha — APPROVED** [review]

**Review by jouni — APPROVED** [review]

**Pull request merged**